manta-benchmark/src/lib.rs

```diff
@@ -49,11 +49,17 @@ impl Context {
     }
 }
 
+impl Default for Context {
+    fn default() -> Self {
+        Self::new()
+    }
+}
```

Comment:
Just add at least `#[derive(Clone, Debug, Default)]` for `Context`. Always prefer to use `derive` than to implement it yourself.

Reply:
Default cannot be derived for Context. We need to explicitly call the Self::new() of Context

Reply:
Yeah you're right sorry, I missed that.